src/types.ts

```diff
@@ -9,6 +9,16 @@ export type AnyDhtHash = HoloHash;
 export type KitsuneAgent = Uint8Array;
 export type KitsuneSpace = Uint8Array;
 
+/** Base64 hash types */
+
+export type HoloHashB64 = string;
```

Comment:
This one is in `src/api/admin/types.ts` too. Please remove it there.

Reply:
Hum I think I removed it? I don't see it in the code...

Reply:
Sorry, I looked at the wrong branch.